app/src/main/java/com/github/se/cyrcle/model/report/ReportedObjectRepository.kt

```diff
@@ -1,5 +1,7 @@
 package com.github.se.cyrcle.model.report
 
+import com.google.firebase.firestore.DocumentReference
```

Comment:
As mentioned above, this import should not exist in the Interface...

Reply:
I will fix this, no worries